src/gui/plugins/view_angle/ViewAngle.qml

```diff
@@ -182,5 +183,130 @@ ToolBar {
         ViewAngle.OnAngleMode(0, 0, 1)
       }
     }
+
+    // set camera pose
+    // xyz
+    Text {
+      Layout.columnSpan: 2
+      Layout.row: 0
+      Layout.column: 5
+      color: "dimgrey"
+      font.bold: true
+      text: "Position (m)"
+    }
+
+    Text {
+      text: "X"
+      color: "dimgrey"
+      Layout.row: 1
+      Layout.column: 5
+    }
+    IgnSpinBox {
+      id: x
+      Layout.row: 1
+      Layout.column: 6
+      value: ViewAngle.camPose[0]
+      maximumValue: 1000.00
+      minimumValue: -1000.00
+      decimals: 2
+      stepSize: 0.01
+      onEditingFinished: ViewAngle.SetCamPose(x.value, y.value, z.value, roll.value, pitch.value, yaw.value)
+    }
+    Text {
+      text: "Y"
+      color: "dimgrey"
+      Layout.row: 2
+      Layout.column: 5
+    }
+    IgnSpinBox {
+      id: y
+      Layout.row: 2
+      Layout.column: 6
+      value: ViewAngle.camPose[1]
+      maximumValue: 1000.00
+      minimumValue: -1000.00
+      decimals: 2
+      stepSize: 0.01
+      onEditingFinished: ViewAngle.SetCamPose(x.value, y.value, z.value, roll.value, pitch.value, yaw.value)
+    }
+    Text {
+      text: "Z"
+      color: "dimgrey"
+      Layout.row: 3
+      Layout.column: 5
+    }
+    IgnSpinBox {
+      id: z
+      Layout.row: 3
+      Layout.column: 6
+      value: ViewAngle.camPose[2]
+      maximumValue: 1000.00
+      minimumValue: -1000.00
+      decimals: 2
+      stepSize: 0.01
+      onEditingFinished: ViewAngle.SetCamPose(x.value, y.value, z.value, roll.value, pitch.value, yaw.value)
+    }
+
+    // rpy
+    Text {
+      Layout.columnSpan: 2
+      Layout.row: 0
+      Layout.column: 7
+      color: "dimgrey"
+      font.bold: true
+      text: "Rotation (rad)"
+    }
+
+    Text {
+      text: "Roll"
+      color: "dimgrey"
+      Layout.row: 1
+      Layout.column: 7
+    }
+    IgnSpinBox {
+      id: roll
+      Layout.row: 1
+      Layout.column: 8
+      value: ViewAngle.camPose[3]
+      maximumValue: 6.28
+      minimumValue: 0.00
```

Comment:
I think we can accept negative rotation values

Reply:
0cdb38b3fdf9d3bebd4c2cc4a3d362103453de07